src/content/nft/index.md

```diff
@@ -305,67 +304,27 @@ Decentralized meaning you and everyone else can verify you own something. All wi
 
-#### The work in minting your NFT {#minting-nfts}
+#### Minting NFTs {#minting-nfts}
 
 When you mint an NFT, a few things have to happen:
 
 - It needs to be confirmed as an asset on the blockchain.
 - The owner's account balance must be updated to include that asset. This makes it possible for it to then be traded or verifiably "owned".
-- The transactions that confirm the above need to be added to a block and "immortalised" on the chain.
-- The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximise their earnings.
+- The transactions that confirm the above need to be added to a block and "immortalized" on the chain.
+- The block needs to be confirmed by everyone in the network as "correct". This consensus removes the need for intermediaries because the network agrees that your NFT exists and belongs to you. And it's on chain so anyone can check it. This is one of the ways Ethereum helps NFT creators to maximize their earnings.
 
-All these tasks are done by miners. And they let the rest of the network know about your NFT and who owns it. This means mining needs to be sufficiently difficult, otherwise anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership. There are lots of incentives in place to make sure miners are acting honestly.
+All these tasks are done by block producers and validators. Block proposers add your NFT transaction to a block and broadcast it to the rest of the network. Validators check that the transaction is valid and then add it to their databases. There are lots of crypto-economic incentives in place to make sure validators are acting honestly. Otherwise, anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership.
 
-[More on mining](/developers/docs/consensus-mechanisms/pow/)
+#### NFT security {#nft-security}
 
-#### Securing your NFT with mining {#securing-nfts}
+Ethereum's security comes from its consensus mechanism which uses proof-of-stake. Validators deposit 32 ETH into a contract that can be destroyed if they misbehave. The system is designed to strongly economically disincentivize malicious actions, making Ethereum effectively tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ETH to change it. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT and the bad actor would be economically penalized and ejected from the pool of validators to prevent it from happening.
```

Comment:
I don't think we need to talk about validators depositing ETH here to make this point

```suggestion
Ethereum's security comes from proof-of-stake. The system is designed to economically disincentivize malicious actions, making Ethereum tamper-proof. This is what makes NFTs possible. Once the block containing your NFT transaction becomes finalized it would cost an attacker millions of ETH to change it. Anyone running Ethereum software would immediately be able to detect dishonest tampering with an NFT, and the bad actor would be economically penalized and ejected.
```